### ISSUE TITLE
lazyImageReady not firing

### ISSUE THREAD

**Issue by pierissimo** [nolimits4web/Swiper#1170]
Hi!
Thanks you again for swiper, v3 rocks.

I would to start using lazyload feature, but the lazyImageRead event seems not to fire (lazyImageLoad instead, yes).
My code is quite simple:

```javascript
var imagesSliderInstance = new Swiper ( $imagesSlider.get(0) , {
        // Optional parameters
        //loop: true,
        preloadImages: false,
        lazyLoading: true,
        onLazyImageReady: function(swiper){
          alert('asd');
        }
      }); 

            <div class="gallery-list-images-slider swiper-container">
			    <div class="swiper-wrapper">
			        <!-- Slides -->

			        <!-- FIRST SLIDE -->
			        <div class="swiper-slide" data-gallery-id="55">
			        	<img data-src="assets/img/samples/home-module-squared-box-with-huge-media.jpg" class="swiper-lazy the-image">
			        	<div class="swiper-lazy-preloader"></div>
			        </div>
				
					<!-- SECOND SLIDE -->
					<div class="swiper-slide" data-gallery-id="55">
						<img data-src="assets/img/samples/home-module-squared-box-with-huge-media.jpg" class="swiper-lazy the-image">
						<div class="swiper-lazy-preloader"></div>
					</div>
			        
			    </div>
			</div>
```

**Comment by nolimits4web** [comment]
Yep, there is an mistake in current dist version, this callback is named `onLazyImageLoaded`, but you can use dev/build version where it is fixed

**Comment by francoismassart** [comment]
@pierissimo 
I did a fork on Swiper's repo in order to be able to use the latest build...

The forked repo:
https://github.com/francoismassart/Swiper

The build folder:
https://github.com/francoismassart/Swiper/tree/master/dist

Dist files via rawgit's CDN:
https://rawgit.com/francoismassart/Swiper/master/dist/js/swiper.jquery.js
https://rawgit.com/francoismassart/Swiper/master/dist/js/swiper.jquery.min.js
https://rawgit.com/francoismassart/Swiper/master/dist/js/swiper.js
https://rawgit.com/francoismassart/Swiper/master/dist/js/swiper.min.js
https://rawgit.com/francoismassart/Swiper/master/dist/css/swiper.css
https://rawgit.com/francoismassart/Swiper/master/dist/css/swiper.min.css

**Comment by nolimits4web** [comment]
Fixed in 3.0.5
